lib/package/kinesis/compileMethodsToKinesis.test.js

```diff
@@ -292,18 +271,14 @@ describe('#compileMethodsToKinesis()', () => {
     )
 
     expect(
-      resource.Properties.Integration.RequestTemplates['application/json']['Fn::Join']
+      resource.Properties.Integration.RequestTemplates['application/json']['Fn::Sub']
     ).to.be.deep.equal([
-      '',
-      [
-        '{',
-        '"StreamName": "',
-        '"undefined"',
-        '",',
-        '"Data": "$util.base64Encode($input.body)",',
-        '"PartitionKey": "$context.requestId"',
-        '}'
-      ]
+      '{"StreamName":"${StreamName}","Data":"${Data}","PartitionKey":"${PartitionKey}"}',
+      {
+        StreamName: 'myStream',
```

Comment:
the original test had `"undefined"` but the new test has `'myStream'`, is that correct?

Reply:
Yes, intentional. Basically, it never reach out here with keeping `undefined` value since it checked in validation phase. And having or not having `StreamName` property itself does not affect this case purpose.

Reply:
same here